src/com/edlplan/ui/fragment/InGameSettingMenu.java

```diff
@@ -116,6 +117,15 @@ private void reload() {
                     .commit();
         });
 
+        enableVideo = findViewById(R.id.enableVideo);
+        enableVideo.setChecked(Config.isEnableVideo());
+        enableVideo.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            Config.setEnableVideo(isChecked);
```

Comment:
I feel like this should be a NOP if the OS version is too low *here*, and maybe with a message saying "Your android version is too old to play video!"